keypress/base.js

```diff
@@ -2,4 +2,44 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-})
+// User hits space, record "start time".
+// Display start time
+//User hits space, record "stop time"
+//Display stop time
+//Calculate stop time - start time
+// Display total time
+
+function millisecondsToTime(milli){
+      var milliseconds = milli % 1000;
+      var seconds = Math.floor((milli / 1000) % 60);
+        if (seconds < 10) {
+          seconds = "0" + seconds;
+        }
+      var minutes = Math.floor((milli / (60 * 1000)) % 60);
+      if (minutes < 10) {
+        minutes = "0" + minutes;
+      }
+      return minutes + ":" + seconds + "." + milliseconds;
+}
+
+var keyPress = false;
```

Comment:
carefull 'overloading' a word (we now have both a variable named keyPress and an event named keypress. It can get confusing).